crates/ruff_python_codegen/src/stylist.rs

```diff
@@ -106,7 +106,13 @@ fn detect_indention(tokens: &[LexResult], locator: &Locator) -> Indentation {
                 }
                 TokenKind::NonLogicalNewline => {
                     let line = locator.line(range.end());
-                    let indent_index = line.chars().position(|c| !c.is_whitespace());
```

Comment:
I think you could do `line.find(|c| !c.is_whitespace())`

Reply:
Wow TIL.